### DISCUSSION TITLE
Does Chia have developer documentation?For example, look up the address, look up the wallet?

### DISCUSSION THREAD

**Discussion by shendejun** [Chia-Network/chia-blockchain#6229] [Support]
Does Chia have developer documentation?For example, look up the address, look up the wallet?

**Comment by BrandtH22** [comment]
@shendejun , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note: all of our documentation for the reference client can be found here https://docs.chia.net/

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
